unit-tests/live/d400/test-pipeline-set-device.py

```diff
@@ -0,0 +1,33 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+# LibCI doesn't have D435i so //test:device D435I// is disabled for now
+# test:device D455
+
+import pyrealsense2 as rs
+from rspy import test
+
+gyro_sensitivity_value = 4.0
+
+with test.closure("pipeline - set device"):
+    cfg = rs.config()
+    ctx = rs.context()
+
+    dev = test.find_first_device_or_exit()
```

Comment:
Note, `find_first_device_or_exit` uses a different context.
I think it's OK. @Nir-Az you see a problem with that?

Reply:
I think it should be OK for this use case